examples/sparseserver-ui/client/settings.py

```diff
@@ -29,17 +31,29 @@ class FeatureHandler:
     variants_desc = "Select your sparse model:"
     variants = {
         "12-Layer BERT Base, Not Sparsified 😢": MultiPipelineClient(
-            model="question_answering/base"
+            model="question_answering/base", address = "deepsparseserver"
         ),
         "12-Layer BERT, Quantized, 99% of Base Accuracy": MultiPipelineClient(
-            model="question_answering/12l_pruned80_quant"
+            model="question_answering/12l_pruned80_quant", address = "deepsparseserver"
         ),
         "6-Layer BERT, Quantized, 96% of Base Accuracy": MultiPipelineClient(
-            model="question_answering/quant6lagg96"
+            model="question_answering/quant6lagg96", address = "deepsparseserver"
         ),
         "3-Layer BERT, Quantized, 89% of Base Accuracy": MultiPipelineClient(
-            model="question_answering/quant3lagg89"
-        ),
+            model="question_answering/quant3lagg89", address = "deepsparseserver"
```

Comment:
`address = "deepsparseserver"` doesn't seem to be needed based on the README update referring to localhost. The address could be settable as a command-line argument for the app if you want to keep that feature. Seems not extensible by hardcoding it to each model

Reply:
Thanks for the review. The deepsparseserver was used due to the internal connectivity via the docker bridge network. 
I've changed the docker network to host in the next PR, so there's no need to keep the feature and it can be referenced directly via localhost.